src/ts/ConvivaAnalytics.ts

```diff
@@ -416,40 +449,42 @@ export class ConvivaAnalytics {
   };
 
   private onPlay = (event: PlaybackEvent) => {
-    this.debugLog('play', event);
+    this.debugLog('[ Player Event ] play', event);
+
     if (this.isAd) {
       // Do not track play event during ad (e.g. triggered from IMA)
+      console.log('[log] returning cause of ad (onplay)');
```

Comment:
is the [log] prefix helpful?
`Ignoring play event because an ad is active` would be easier to understand